res/controllers/Reloop Terminal Mix 2-4.midi.xml

```diff
@@ -1,3344 +1,3561 @@
-                </options>
-            </control>
-            <control> <!-- Trax knob press -->
-                <group>[Playlist]</group>
-                <key>TerminalMix.traxKnobPress</key>
-                <status>0x90</status>
-                <midino>0x39</midino>
-                <options>
-                    <Script-Binding/>
-                </options>
-            </control>
-            <control> <!-- Back button -->
-                <group>[Playlist]</group>
-                <key>TerminalMix.backButton</key>
-                <status>0x90</status>
-                <midino>0x37</midino>
-                <options>
-                    <Script-Binding/>
-                </options>
-            </control>
-            <!-- Cross-fader assign switches -->
-            <control>
-                <group>[Channel1]</group>
-                <key>TerminalMix.cfAssignL</key>
-                <status>0x90</status>
-                <midino>0x3B</midino>
-                <options>
-                    <Script-Binding/>
-                </options>
-            </control>
-            <control>
-                <group>[Channel1]</group>
-                <key>TerminalMix.cfAssignM</key>
-                <status>0x90</status>
-                <midino>0x3C</midino>
-                <options>
-                    <Script-Binding/>
-                </options>
-            </control>
-            <control>
-                <group>[Channel1]</group>
-                <key>TerminalMix.cfAssignR</key>
-                <status>0x90</status>
-                <midino>0x3D</midino>
-                <options>
-                    <Script-Binding/>
-                </options>
-            </control>
-            <control>
-                <group>[Channel2]</group>
-                <key>TerminalMix.cfAssignL</key>
-                <status>0x90</status>
-                <midino>0x3E</midino>
-                <options>
-                    <Script-Binding/>
-                </options>
-            </control>
-            <control>
-                <group>[Channel2]</group>
-                <key>TerminalMix.cfAssignM</key>
-                <status>0x90</status>
-                <midino>0x3F</midino>
-                <options>
-                    <Script-Binding/>
-                </options>
-            </control>
-            <control>
-                <group>[Channel2]</group>
-                <key>TerminalMix.cfAssignR</key>
-                <status>0x90</status>
-                <midino>0x40</midino>
-                <options>
-                    <Script-Binding/>
-                </options>
-            </control>
+<MixxxMIDIPreset mixxxVersion="2.1.0" schemaVersion="1">
+  <info>
+    <name>Reloop Terminal Mix 2/4 2.1</name>
+    <author>Sean M. Pappalardo (1.11), ronso0 (2.1 update)</author>
+    <description>
+A complete 4-deck preset for a single Reloop Terminal Mix 2 or 4.
+Uses scripting. Open script file and adapt variables to your needs:
```

Comment:
Please remove "Uses scripting". This is an implementation detail that should not be presented to users. Also, I do not see the mentioned variables in the script file. Could you remove mention of those too?

Reply:
Please be patient. as I said, this is still in progress. I'm working on it right now, and of course I'll clean up the variables as well